src/main/java/com/datapipe/jenkins/vault/VaultAccessor.java

```diff
@@ -17,8 +19,13 @@
     private transient VaultConfig config;
 
     public void init(String url) {
+        init(url, false);
+    }
+
+    public void init(String url, boolean skipSslVerification) {
         try {
-            config = new VaultConfig().address(url).build();
+            config = new VaultConfig(url).sslVerify(skipSslVerification)
+                    .build();
```

Comment:
Looking at the vault-java-driver code for 3.1.0 (currently in use) This should look like

```
config = new VaultConfig().address(url).sslConfig(new SslConfig().verify(skipSslVerification).build()).build()
```